markdown/cmark/config.h.in

```diff
@@ -0,0 +1,76 @@
+#ifndef CMARK_CONFIG_H
+#define CMARK_CONFIG_H
```

Comment:
this file seems useless in the current setup.  However, some of its content could be moved to Autotools…

Reply:
I don't think I want to care about anything in this file that I can trivially stub-out. Maybe I could just drop it, will have to look closer. Same for most of below comments on this file.